src/client/ujs.ts

```diff
@@ -0,0 +1,20 @@
+import * as UJS from '@rails/ujs';
+
+function customConfirm(message: string, element: HTMLElement) {
```

Comment:
Could you add a short JSdoc explaining what this function does?  It seems to be a custom version of `window.confirm` but I think it'd be helpful to have a quick description